### ISSUE TITLE
dyld: lazy symbol binding failed: Symbol not found

### ISSUE THREAD

**Issue by markmorris** [LinusU/node-appdmg#167]
Hope you can point me in the right direction. 
This only happens when i try and set an Icon when creating a DMG. If theirs no icon, it runs successful with signing.

Node: 8.12
MacOS High Sierra (10.13.6)

UPDATE:
WOULD BE WILLING TO PAY SOMEONE TO FIX THIS ASAP. 

```
dyld: lazy symbol binding failed: Symbol not found: __ZN4node19GetCurrentEventLoopEPN2v87IsolateE
  Referenced from: /Users/Musicline/Desktop/marks stuff/macos-packager/node_modules/fs-xattr/build/Release/xattr.node
  Expected in: flat namespace

dyld: Symbol not found: __ZN4node19GetCurrentEventLoopEPN2v87IsolateE
  Referenced from: /Users/Musicline/Desktop/marks stuff/macos-packager/node_modules/fs-xattr/build/Release/xattr.node
  Expected in: flat namespace
```

Just want to point out, ive already tried most of the obvious things, including delete node_modules and rebuilding, even set this up on a new project and same thing happened.

**Comment by markmorris** [comment]
No one?

**Comment by LinusU** [comment]
I'm sorry, I don't think I can be of much help here. Have you tried upgrading to a newer version of Node.js? Either the latest 8.x, or a 10.x release...

**Comment by markmorris** [comment]
yeah i tried literally all versions available. Kept throwing up the same error in slightly different format. Thanks for the reply, i will investigate more when i have the spare time.

**Comment by matthewnogas** [comment]
Any progress with this? I'm having the same error

**Comment by markmorris** [comment]
Sorry no, i'm still getting the error if i try and change the icon.

**Comment by agathver** [comment]
It's unfortunate that the NodeJS folks broke API in between releases. There is nothing much we can do in this regard. You can try `npm rebuild fs-xattr`

**Comment by simfer** [comment]
I fixed it with `npm rebuild bcrypt --build-from-source` as reported here: https://github.com/kelektiv/node.bcrypt.js/issues/16

**Comment by LinusU** [comment]
I've converted node-xattr to use NAPI bindings instead of the v8 ones, so this should be resolved now!

Fixed in [🚢 0.6.0](https://github.com/LinusU/node-appdmg/releases/tag/v0.6.0)
